tests/test_open_file.py

```diff
@@ -47,9 +47,19 @@ def test_file_has_correct_types():
     assert types == [1200, 1212, 1201, 1202, 1203, 1204, 1209, 1210]
 
 
-def test_types_gzipped():
-    testfile = 'tests/resources/one_shower.dat'
-    f = eventio.EventIOFile(testfile)
+def test_types_gzip():
+    testfile = 'tests/resources/one_shower.dat.gz'
+    f = eventio.EventIOFile(testfile, zcat=False)
     types = [o.header.type for o in f]
 
     assert types == [1200, 1212, 1201, 1202, 1203, 1204, 1209, 1210]
+
+
+def test_types_zcat():
+    from eventio.base import PipeWrapper
+    testfile = 'tests/resources/one_shower.dat.gz'
+    f = eventio.EventIOFile(testfile)
+
+    assert isinstance(f._filehandle, PipeWrapper)
```

Comment:
Ah nice way to test this ... I was not sure how to find out if zcat was really used, due to the falling back to gzip behaviour... but this is nice ...